build.gradle

```diff
@@ -88,7 +89,7 @@ subprojects {
   }
 
   test {
-    maxHeapSize = "512m"
+    maxHeapSize = "1024m"
```

Comment:
i will change it back to 512m, but on my machine it won't work unless I increase it to 1024m.